GVFS/GVFS.UnitTests.Windows/Windows/Upgrader/UpgradeOrchestratorWithIProductUpgraderTests.cs

```diff
@@ -0,0 +1,202 @@
+﻿using GVFS.Common;
```

Comment:
Why is this test in `GVFS/GVFS.UnitTests.Windows/` and not `GVFS/GVFS.UnitTests`?

Can we run these tests on both platforms?

Reply:
Agree that `GVFS/GVFS.UnitTests` is the rightful place for upgrader tests. But right now Upgrade is supported only on Windows. Both the upgrade paths today (Github & Nuget) are Windows only. We can move the tests when we port upgrader to Mac.

Reply:
Just a small clarification - `UpgradeOrchestrator` is part of `GVFS.Upgrader`, which is .NET framework (Windows) only.

The actual upgrader implementations are in `GVFS.Common` and those can be tested in `GVFS.UnitTests`, unless they are tested via the `UpgradeOrchestrator`

Reply:
> Just a small clarification - UpgradeOrchestrator is part of GVFS.Upgrader, which is .NET framework (Windows) only.

> The actual upgrader implementations are in GVFS.Common and those can be tested in GVFS.UnitTests, unless they are tested via the UpgradeOrchestrator

Ok thanks!